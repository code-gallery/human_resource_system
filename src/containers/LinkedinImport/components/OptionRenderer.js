import React from 'react'
import PropTypes from 'prop-types'

const OptionRenderer = (props) => {
  return (
    <div className="institution-select">
      <img src={props.logo_image ? props.logo_image : '/images/appii-placeholder.png'} alt={props.name} />
      <p>{props.name}</p>
      <span>{props.country}</span>
    </div>
  )
}

OptionRenderer.propTypes = {
  logo_image: PropTypes.string,
  name: PropTypes.string,
  country: PropTypes.string
}

export default OptionRenderer
