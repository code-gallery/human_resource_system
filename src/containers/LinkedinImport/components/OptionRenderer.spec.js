import React from 'react'
import renderer from 'react-test-renderer'
import OptionRenderer from './OptionRenderer'

describe('containers/LinkedinImport/OptionRenderer', () => {
  it('Renders correctly', () => {
    const wrapper = renderer.create(
      <OptionRenderer
        logo_image="test.jpg"
        name="Test name"
        country="Test country"
      />
    ).toJSON()

    expect(wrapper).toMatchSnapshot()
  })
})

