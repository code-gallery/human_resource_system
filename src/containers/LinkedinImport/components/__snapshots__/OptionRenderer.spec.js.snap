// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`containers/LinkedinImport/OptionRenderer Renders correctly 1`] = `
<div
  className="institution-select"
>
  <img
    alt="Test name"
    src="test.jpg"
  />
  <p>
    Test name
  </p>
  <span>
    Test country
  </span>
</div>
`;
